pkg/render/fluentd.go

```diff
@@ -732,7 +760,7 @@ func (c *fluentdComponent) envvars() []corev1.EnvVar {
 
 	if c.SupportedOSType() != rmeta.OSTypeWindows {
 		envs = append(envs,
-			corev1.EnvVar{Name: "CA_CRT_PATH", Value: c.cfg.TrustedBundle.MountPath()},
+			corev1.EnvVar{Name: "CA_CRT_PATH", Value: TigeraCertBundleMountPath},
```

Comment:
Syslog can be used on windows hosts, so you also need to add the env now for windows.

Reply:
Looks like [this code](https://github.com/tigera/operator/blob/62170a80071ec2bac8822612950c1c5ef55bf1f6/pkg/render/fluentd.go#L733) is intent to populate the EnvVar for !Windows usecase.  Also "[Should render for windows nodes" ](https://github.com/tigera/operator/blob/62170a80071ec2bac8822612950c1c5ef55bf1f6/pkg/render/fluentd_test.go#L193) UT fails if we populate these values for Windows usecase

Reply:
Updated

Reply:
DOne